pom.xml

```diff
@@ -66,9 +67,20 @@
 
     <dependencies>
         <dependency>
+            <groupId>org.slf4j</groupId>
+            <artifactId>slf4j-api</artifactId>
+            <version>1.7.18</version>
+        </dependency>
+        <dependency>
             <groupId>org.cometd.java</groupId>
             <artifactId>cometd-java-client</artifactId>
             <version>3.0.9</version>
+            <exclusions>
```

Comment:
not sure you need this exclusion since you are being explicit about what version of slf4j-api you want in the above dependency

Reply:
Won't compile w/out the exclusion

Reply:
👍